docs/articles/new-expensify/connections/Set-up-QuickBooks-Online-connection.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Set up QuickBooks online connection
+description: Integrate QuickBooks Online with Expensify
+---
+<div id="new-expensify" markdown="1">
+
+{% include info.html %}
+To use the QuickBooks Online connection, you must have a QuickBooks Online account and an Expensify Collect plan. The QuickBooks Self-employed subscription is not supported.
+{% include end-info.html %}
+
+The features available for the Expensify connection with QuickBooks Online vary based on your QuickBooks subscription. The features may still be visible in Expensify even if you don’t have access, but you will receive an error if the feature isn't available with your subscription. 
+
+Here is a list of the features supported by each QuickBooks Online subscription: 
+
+| Feature                    | Simple Start | Essentials | Essentials Plus |
+|----------------------------|--------------|------------|-----------------|
+| Expense Reports            | &#10004;     | &#10004;   | &#10004;        |
+| GL Accounts as Categories  | &#10004;     | &#10004;   | &#10004;        |
+| Credit Card Transactions   | &#10004;     | &#10004;   | &#10004;        |
+| Debit Card Transaction     |              | &#10004;   | &#10004;        |
+| Classes                    |              | &#10004;   | &#10004;        |
+| Customers                  |              | &#10004;   | &#10004;        |
+| Projects                   |              | &#10004;   | &#10004;        |
+| Vendor Bills               |              | &#10004;   | &#10004;        |
+| Journal Entries            |              | &#10004;   | &#10004;        |
+| Tax                        |              | &#10004;   | &#10004;        |
+| Billable                   |              |            | &#10004;        |
+| Location                   |              |            | &#10004;        |
+
+To set up your QuickBooks Online connection, complete the 5 steps below.
+
+# Step 1: Set up employees in QuickBooks Online
+
+Log in to QuickBooks Online and ensure all of your employees are setup as either Vendors or Employees using the same email address that they are listed under in Expensify. This process may vary by country, but you can go to **Payroll** and select **Employees** in QuickBooks Online to add new employees or edit existing ones. 
+
+# Step 2: Connect Expensify to QuickBooks Online
+
+<ol type="a">
+   <li>Click your profile image or icon in the bottom left menu.</li>
+   <li>Scroll down and click <b>Workspaces</b> in the left menu.</li>
+   <li>Select the workspace you want to book the travel under.</li>
```

Comment:
This travel reference feels out of place

Reply:
It totally was! EXCELLENT CATCH—thank you!! I updated it to the correct phrasing